src/APICore.ts

```diff
@@ -23,8 +23,8 @@ export default class API {
 
     constructor(private config: APIConfiguration) {}
 
-    async get<T>(url: string, queryParams?: any, args: RequestInit = {method: 'get'}): Promise<T> {
-        return await this.request<T>(url + convertToQueryString(queryParams), args);
+    async get<T>(url: string, args: RequestInit = {method: 'get'}): Promise<T> {
```

Comment:
I think I liked the old design better :/

The second parameter was simplifying the way `get` requests were send. Now you have to know and remember that you must build your own path *only* for get requests

Reply:
If you check #14 (src/resources/Groups/Groups.ts) you'll see that you also might need to use the `buildPath` for other requests since the backend is designed this way

Now at least the behaviour is consistent between each methods

Reply:
All the other requests (`post`, `postForm`, `put`, `delete`) didn't have a `queryParams` parameter, so now the way to make requests is standard for all methods.

Adding a `queryParams` to all other methods wasn't an option because we already had 3 parameters and generally its a bad idea to have more than 3 params.